project/Header.scala

```diff
@@ -1,27 +1,27 @@
-import sbt._
-import Keys._
+// import sbt._
```

Comment:
Can you open an issue to keep track of this? We will need to enable it after RC is out and sbt-header cross-compiles.